sbt-scalajs-bundler/src/main/scala/scalajsbundler/Stats.scala

```diff
@@ -115,13 +115,13 @@ object Stats {
   )(Asset.apply _)
 
   implicit val errorReads: Reads[WebpackError] = (
```

Comment:
@ptrdom what do you think of just using semi-auto derivation in `Stats.scala`?:

```scala
  implicit val assetsReads: Reads[Asset] = Json.reads[Asset]
  implicit val errorReads: Reads[WebpackError] = Json.reads[WebpackError]
  implicit val warningReads: Reads[WebpackWarning] = Json.reads[WebpackWarning]
  implicit val pathReads: Reads[Path] = Reads.StringReads.map(new File(_).toPath)
  implicit val statsReads: Reads[WebpackStats] = Json.using[Json.WithDefaultValues].reads[WebpackStats]
```

I saw this used in https://github.com/scalacenter/scalajs-bundler/pull/408 and tested this works and saves a few lines of code ;)